requirements.yml

```diff
@@ -1,6 +1,4 @@
 ---
 collections:
-  - name: kubernetes.core
```

Comment:
i thought we need this collection? how are we expected to call `kubernetes.core` without it?

I get a collection missing error when it's not there.

Reply:
The fips-assessment collection includes it as a dependency. So, we don't need to. But then again, since we do call kubernetes.core directly, it should probably be left in.